xbmc/Application.cpp

```diff
@@ -211,7 +211,7 @@
 #if defined(TARGET_ANDROID)
 #include "platform/android/activity/XBMCApp.h"
 #include "platform/android/activity/AndroidFeatures.h"
-#include "platform/android/jni/Build.h"
+#include "androidjni/Build.h"
```

Comment:
Kodi explicitly adds the depends folder to gcc's -I flag. IIRC using &lt;> is only needed for implicitly-added folders, like system folders.